src/components/m-accordion-item/_preview.html

```diff
@@ -0,0 +1,115 @@
+<axa-accordion-item icon="checkmark" classes="m-accordion-item" header="Estate Protection Rider" header-secondary="if both you and your spouse pass away, this rider provides your beneficiaries with an additional death benefit" header-color="blue" multiple>
```

Comment:
@TheDadi Please hardcode all needed CSS classes inside the Webcomponent itself and remove that `classes="m-accordion-item"` attribute.